build.sbt

```diff
@@ -129,21 +129,34 @@ lazy val scalaVersionDependentSettings = Def.settings(
     if (scalaVersion.value.startsWith("2."))
       scalacOptions.value
     else
-      scalacOptions.value.filterNot(Set("-Ywarn-macros:after", "-Vimplicits", "-Vtype-diffs"))
+      scalacOptions
+        .value
+        .filterNot(Set("-Ywarn-macros:after", "-Vimplicits", "-Vtype-diffs", "-Wconf:any:warning-verbose"))
```

Comment:
overlooked "-Wconf:any:warning-verbose" in the previous PR.

Reply:
That should work on **Scala 3**, shouldn't it?

Reply:
Ah, it found it works after checking again.

Reply:
Thanks.